src/components/shared/CallNodeContextMenu.js

```diff
@@ -22,6 +22,10 @@ import {
   getImplementationFilter,
   getInvertCallstack,
 } from '../../selectors/url-state';
+import {
+  getRightClickedCallNodePath,
+  getRightClickedCallNodeIndex,
+} from '../../selectors/right-clicked-call-node-path';
```

Comment:
I was wondering if this would be in its own file or moved to an existing file... but this is good like this :)

Reply:
I wasn't sure if I wanted to move it all in a separate file and was thinking about keeping it with the `profile` selectors at first as this is where `getRightClickedTrack` is localed, but this was causing circular dependency with `per-thread` selectors 😅